client/src/components/ElementEditor/ElementList.js

```diff
@@ -74,9 +74,9 @@ class ElementList extends Component {
     if (!isDraggingOver) {
       output = [
         <HoverBarComponent
-          key={0}
+          key={`0_${areaId}`} // Make key uniq
```

Comment:
We need a key here because the element is in an array. The key just needs to be unique within that array ... it doesn't need to be unique within the page.

Although it doesn't hurt for it to be unique with the page.

Reply:
DONE